packages/openapi-generator/src/parser/schema.ts

```diff
@@ -169,9 +169,17 @@ function parseEnumSchema(
   const type = schema.type ? getType(schema.type) : 'string';
   return {
     type,
-    enum: (schema.enum || []).map(entry =>
-      type === 'string' ? getEnumStringValue(entry) : entry
-    )
+    enum: (schema.enum || []).map(entry => {
+      if (type === 'string' && entry !== null) {
+        return getEnumStringValue(String(entry));
+      }
+      if (entry === null && !schema.nullable) {
+        throw new Error(
+          'null was used as a parameter in an enum, although the schema was not declared as nullable'
```

Comment:
I'll just repeat my suggestion ;)

Should we be this restrictive? Shouldn't we rather log a warning, but still add null as a possible value in the enum? @FrankEssenberger @jjtang1985 what are your opinions?

Reply:
I'm fine to use warning instead of error for enabling the user. The worst case would be, the user receive an error from server side and stop using `null`. For that we can argue, the spec is anyway confusing and needs some actions from the service owner.

Reply:
I added a comment above